modules/all-modules/pom.xml

```diff
@@ -47,16 +47,6 @@
         </dependency>
         <dependency>
             <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-file-extension-common</artifactId>
```

Comment:
You are right, it is maintained. The thing is this dependency was duplicated that's why I'm removing it here. Line 130 has the same dependency declared.